radix-engine-interface/src/api/kernel_modules/auth_api.rs

```diff
@@ -0,0 +1,8 @@
+use crate::{api::types::*, blueprints::resource::AccessRule};
+use sbor::rust::fmt::Debug;
+
+pub trait ClientAuthApi<E: Debug> {
+    fn get_auth_zone(&mut self) -> Result<ObjectId, E>;
```

Comment:
Interesting approach, do you see this as the long term approach to getting the object id? If so, does it make sense to have our other "transient" methods to use the same approach?

Reply:
I found it the only sensible way of exposing nodes created by kernel module. We can try to extend to other methods if node abstraction is needed.